CONTRIBUTING.md

```diff
@@ -1,52 +1,160 @@
 
-- [gliff.ai](https://gliff.ai) tells you who we are
-- [Our roadmap](https://github.com/gliff-ai/roadmaplist) tells you what we're thinking about right now and in the future
-- [gliff.ai Projects](https://github.com/orgs/gliff-ai/projects) is where we keep our day-to-day project management
-- Bugs or vulnerabilities? [Issues](https://github.com/gliff-ai/gliff-ai/issues) is where to report them
-  - We have some [guidelines](https://github.com/gliff-ai/.github/SECURITY.md) for reporting vulnerabilities
-  - And some templates for issues that we'd appreciate you using
+Each repository has one of two topic tags to help you and fellow contributors understand which repositories [gliff.ai](https://gliff.ai) is open to contributions outside of our team. They are:
+
+- **contributors-welcome** – We welcome contributions on this repository!
+- **no-contributors** – This repository does not currently accept contributions unfortunately!
+
+👍🎉 **Thank you from the gliff.ai team for your interest and hopefully taking the time to contribute!** 🎉👍
+
+## Table of Contents
+
+Looking for something specific? 🔍
+
+- [Introduction](#gliffai-contribution-guide)
+- [Code of Conduct](#code-of-conduct)
+- [Create an Issue](#create-an-issue)
+- [Create a Pull Request](#create-a-pull-request)
+- [Linting, Formatting and Testing](#linting-formatting-and-testing)
+- [Commit Best Practices and Naming Conventions](#commit-best-practices-and-naming-conventions)
+- [Assigning Reviewers Best Practices](#assigning-reviewers-best-practices)
+- [Labels](#labels)
+- [Contact](#contact)
+
+## Code of Conduct
+
+[{{back to navigation}}](#table-of-contents)
+
+All projects and contributors are governed by the [gliff.ai Code of Conduct](https://github.com/gliff-ai/.github/blob/main/CODE_OF_CONDUCT.md) 📝. By contributing, you are expected to uphold this code to ensure a positive and welcome community space for all.
+
+Please report unacceptable behavior ⚠️ to the gliff.ai team at [contact@gliff.ai](mailto:contact@gliff.ai?subject=Report%20GitHub%20Behavior) or on GitHub.
+
+## Create an Issue
+
+[{{back to navigation}}](#table-of-contents)
+
+Our team uses 4 templates for creating issues across gliff.ai repositories, and we would appreciate you also using these and filling them out with as much detail as you are able to provide. The templates include:
+
+- **Bug Report** – report any bugs whether technical or visual (the “bug” label will be automatically applied). 
+- **Feature/Enhancement Request** – request any new features or enhancements of current features (the “enhancement” label will be automatically applied).
```

Comment:
Should we actually have a separate "New feature" vs "Enhancement" process?

Reply:
I had this thought as it would be worth that extra organisation level with another label. I'll get on it separating this into two issue templates and auto-assigned labels!